client-mvp-04/src/utils/uiKit/uiKit.js

```diff
@@ -162,6 +162,28 @@ const DevUiKit = () => {
       </div>
       <p className={'dev-comment'}>Input with icon</p>
 
+      <label for="big-radio-button" className={'radio-button-container'}>
+        <input
+          type="radio"
+          name="big-radio-button"
+          id="big-radio-button"
+          className={'actual-radio-button'}
+        />
+        <div className={'custom-radio-button'}></div>
```

Comment:
XML tag has an empty body
Replace with `<div className={'custom-radio-button'} />`
And it`s better to replace the div tag with span because a label is an inline element